app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -60,7 +58,7 @@ def get_feature(self, obj):
 
         return {
             "featureId": feature_config_slug,
-            "enabled": feature_enabled,
+            "enabled": True,
```

Comment:
I would suggest to add a note as todo that we are setting this value as True and this later will get removed when we no longer support 104 version